lang/java/reef-bridge-proto-java/src/main/java/org/apache/reef/bridge/parameters/DriverServicePortRangeCount.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.bridge.parameters;
+
+import org.apache.reef.tang.annotations.Name;
+import org.apache.reef.tang.annotations.NamedParameter;
+
+/**
+ * The port range count.
+ */
+@NamedParameter(doc = "Driver service port range count from start port",
+    short_name = "port-range-count", default_value = "100")
+public final class DriverServicePortRangeCount implements Name<Integer> {
```

Comment:
Can we please use the `PortProvider` mechanism for this? That way, we only need to specify ranges once per container.

Reply:
thanks for making me aware of that mechanism. I will switch over.